fili-core/src/main/java/com/yahoo/bard/webservice/web/FiliResponseWriterSelector.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Response writer selector object.
```

Comment:
This is the place where I would specify that this selector selects the `ResponseWriter` based on the format type in the `ApiRequest`, rather than on the constructor or method signature.